scripts/build-version.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const path = require("path");
+
+const zero = digit => `0${digit}`.slice(-2);
+
+const versionDate = date => {
+  const date_ = `${date.getFullYear()}-${zero(date.getMonth() + 1)}-${zero(
+    date.getDate(),
+  )}`;
+  const time = `${zero(date.getHours())}-${zero(date.getMinutes())}-${zero(
+    date.getSeconds(),
+  )}`;
+  return `${date_}-${time}`;
+};
+
+const now = new Date();
+
+const data = JSON.stringify(
+  {
+    app: `excalidraw-${versionDate(now)}.zip`,
+    version: versionDate(now),
```

Comment:
```suggestion
    version: now.toISOString().split('T')[0],
```

Reply:
Also, does this mean there will only be one update per day?

Reply:
that's why I included hour/min/sec

Reply:
Ah — why not use the iso8601 format?